### ISSUE TITLE
code examples - remove $ prompts at the start

### ISSUE THREAD

**Issue by yochannah** [intermine/im-docs#36]
in #35, we added a copy/paste button to the code examples throughout the docs - this is awesome, and now every code example is a click away from being copied. 

The only problem - we often have `$` in front of code examples, with the intent of indicating to the reader that this is a command prompt. When copy/pasting manually they would have had to manually remove the $ each time, which was perhaps a bit annoying - but with the copy/paste button we now have, the `$` really needs to go.

Also, some of the code examples have filepaths. These should be deleted too. Example: 

docs page: https://intermine.readthedocs.io/en/latest/get-started/tutorial/  
code: https://github.com/intermine/im-docs/blob/master/docs/get-started/tutorial/index.rst

```bash
$ mkdir git
$ cd git
~/git $ git clone https://github.com/intermine/biotestmine
```

should become... 
```bash
mkdir git
cd git
git clone https://github.com/intermine/biotestmine
```

## if you want to pick up this task

0. Take a look at the [InterMine contributing guidelines](http://intermine.org/contributing/)
1. Comment on this issue stating that you intend to work on the task
2. When you're ready, add your work to the repo and create a pull request.

### What to do if you need help

Mention @yochannah, tweet [@yoyehudi](http://www.twitter.com/yoyehudi), pop by to say hi on [chat](http://chat.intermine.org) or if needed email yo@intermine.org. Don't forget we're usually only available during uk office hours and will not be able to respond at other times :)

**Comment by Igoranze** [comment]
Intend to work on this issue

**Comment by yochannah** [comment]
awesome - let me know if you have any questions! :D

**Comment by Igoranze** [comment]
Sure thing!
I do have a question about the file paths, are these consistent always before the `$` ?

**Comment by yochannah** [comment]
I'm afraid I'm not actually sure, sorry! It was written several years ago,
before I started working on the project. If some are missed, though, it's
not the end of the world - we'll still be better off than we were before we
removed them!

**Comment by Igoranze** [comment]
Hi, sorry to bother you again @yochannah. I'm currently working on this issue but my intention is to create a new enhanced `copybutton.js` file which triggers when the copy button is pressed.
This new `copybutton.js` file contains code that removes the `$` tag and everything that comes before the `$`. So visually `$` would still be in the documents, but when the copy code button is pressed only the "code" without the `$` and without the file paths (which come before `$`).

But my guess is you would just want the *.rst files updated with the file paths and `$` removed?

So my question is, do you have a preference?

**Comment by yochannah** [comment]
I think - so long as it works okay with readthedocs - either approach
should be fine!

**Comment by Officialkome** [comment]
Hello @yochannah if this issue is still open, I would like to try editing this tutorial manually.

Please assign to me. Thank you

**Comment by Officialkome** [comment]
Hello @yochannah if this issue is still open, I would like to try editing this tutorial manually. Do I need to know Perl to do this? 

Please let me know if I can work on this. Thank you

**Comment by CipherGirl** [comment]
I'd like to pick this up.. I have basics in sphinx..

**Comment by uosl** [comment]
@Officialkome I apologise that we overlooked your comment.

This issue has been now fixed with #37.
